react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/RNPaywallsModule.kt

```diff
@@ -33,13 +34,10 @@ internal class RNPaywallsModule(reactContext: ReactApplicationContext) :
 
     @ReactMethod
     fun presentPaywall(
-        options: ReadableMap,
```

Comment:
I think typed parameters are safer so I replaced the map with the real parameters now that we don't need a nullable bolean `displayCloseButton` in iOS, which was what made me add this map parameter